### PR TITLE
:sparkles: Added contentMaxHeight prop to Dialog

### PR DESCRIPTION
**Pull request by mariush2**
- [AB#558763](https://dev.azure.com/Equinor/1f7078da-0ba3-4461-a220-c3e39c5c1f09/_workitems/edit/558763)

### CODE REVIEW

File: src/molecules/Dialog/Dialog.stories.tsx [hunk 1]
```diff
@@ -223,3 +223,52 @@ export const ColorVariations: Story = {
     ],
   },
 };
+
+export const LongDialogWithContentMaxHeight: Story = {
+  args: {
+    title: 'Left and right dialog!',
```
Comment (aslakihle) [inline src/molecules/Dialog/Dialog.stories.tsx:229]: Ultra nitpick c:
Should this title be Long dialog with ContentMaxHeight? :D

**Review by aslakihle — COMMENTED** [review]
lil nitpick 😏

File: src/molecules/Dialog/Dialog.stories.tsx (hunk 1 above)
Comment (mariush2) [reply src/molecules/Dialog/Dialog.stories.tsx]: Pushed a commit

**Comment by mariush2** [comment]
Included a version bump as well to get this out to the Premo team

**Review by aslakihle — APPROVED** [review]

**Pull request merged**